### ISSUE TITLE
Upgrade DiffusionBee dependancies to compile ES2020

### ISSUE THREAD

**Issue by joshJarr** [web3-storage/diffusionbee-stable-diffusion-ui#10]
see: https://github.com/divamgupta/diffusionbee-stable-diffusion-ui/pull/299

-----

> 
> When working with the electron app we noticed ES2020 was not being compiled correctly. Features such as [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) were throwing compilation errors as the loader for these files were outdated. This was also the case for files imported from node_modules meaning any library that used this modern JS syntax would not compile.
> 
> To fix this we updated the vue-cli-plugins and noticed that [babel-eslint](https://www.npmjs.com/package/babel-eslint) is deprecated, so updated this to use[ @babel/eslint-parser](https://www.npmjs.com/package/@babel/eslint-parser).
> 
> This had a side effect of also upgrading webpack to version 5+.
> [Webpack 5+ no longer polyfills node.js](https://webpack.js.org/blog/2020-10-10-webpack-5-release/#automatic-nodejs-polyfills-removed) modules, this caused a few errors across the frontend so we added [node-polyfill-webpack-plugin](https://www.npmjs.com/package/node-polyfill-webpack-plugin) to ensure these necessary modules were available in the frontend.
> 
> A side effect of bumping the babel and vue-cli versions were that automated linting occurred when building and serving the electron app blocking compilation. To get around this we added a temporary .eslintignore file to discard these complaints for now.
> 
> TLDR: This PR updates the vue-cli and babel/eslint dependancies so that they are able to compile JS files and modules that use modern ES2020 syntax, aligning with the vue (.vue) file loader.
